lnwallet/btcwallet/btcwallet.go

```diff
@@ -705,7 +718,7 @@ out:
 			// notifications for any newly unconfirmed transactions.
 			go func() {
 				for _, tx := range txNtfn.UnminedTransactions {
-					detail, err := unminedTransactionsToDetail(tx)
+					detail, err := unminedTransactionsToDetail(tx, t.w.ChainParams())
```

Comment:
Line seems to exceed 80 character limit? If it does, it should be formatted like so:

```
unminedTransactionsToDetail(
    tx, tx.w.ChainParams(),
)
```